### PR TITLE
Add frv

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds a recipe for `Frv` a library used in gravitational-wave astrophysics.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, the windows build here is failing (I think) because the wrong `pkg-config` executable is being used. From [the cmake output](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=297126&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=708):

```
-- Found PkgConfig: C:/Strawberry/perl/bin/pkg-config.bat (found version "0.26") 
```

What's the easiest fix for this?

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, after working around the pkg-config-related windows failure above, this is now ready for review, thanks.

**Comment by duncanmmacleod** [comment]
Ping @conda-forge/staged-recipes, is someone able to review and merge this recipe?

File: recipes/frv/build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	-DCMAKE_BUILD_TYPE:STRING=Release \
+	-DCMAKE_INSTALL_LIBDIR:PATH="lib" \
+	-DCMAKE_INSTALL_PREFIX:PATH=${PREFIX} \
+	${SRC_DIR}
+
+# build
+cmake --build . --parallel ${CPU_COUNT} --verbose
+
+# install
+cmake --build . --parallel ${CPU_COUNT} --verbose --target install
```
Comment (ocefpaf) [inline recipes/frv/build.sh:18]: Is there a `ctest` or make check/test we ca run here?

File: recipes/frv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{% set name = "Frv" %}
+{% set version = "v4r35p6" %}
+
+# handle Virgo version number format
+{% set semver_version = version|
+    replace('v', '', 1)|
+    replace('r', '.', 1)|
+    replace('p', '.', 1)
+ %}
```
Comment (ocefpaf) [inline recipes/frv/meta.yaml:9]: Ouch!

**Comment by rhyspoulton** [comment]
I am happy to be listed as a maintainer of this recipe.

File: recipes/frv/build.sh (hunk 1 above)
Comment (duncanmmacleod) [reply recipes/frv/build.sh:18]: @ocefpaf, not yet, but we can lean on the developers to add a suite.

File: recipes/frv/meta.yaml (hunk 2 above)
Comment (duncanmmacleod) [reply recipes/frv/meta.yaml:9]: Indeed, this seems to be a legacy of the CMT build system used in high-energy physics back in the 90s.

**Comment by fcarbogn** [comment]
I am also happy to be listed as a maintainer of this recipe.

**Pull request merged**